src/main/java/uk/gov/hmcts/reform/sscs/service/hmc/topic/ProcessHmcMessageServiceFactory.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.hmcts.reform.sscs.service.hmc.topic;
+
+import java.util.List;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Slf4j
+@Service
+public class ProcessHmcMessageServiceFactory {
```

Comment:
I don't think we need this factory as only instance of the service will be enabled based on the property.

Reply:
done